### PR TITLE
fix: CSS "content:"

### LINKED ISSUE #4984: [BUG] Strange parentheses in feed management in Nord theme
**Describe the bug**

In Nord theme there are parentheses behind feed names that aren't there in Origine.

**To Reproduce**
1. Go to 'Subscription management'

**Expected behavior**
No parentheses.

**Screenshots**
image

**Environment information (please complete the following information):**
- Device: laptop
- OS: Ubuntu 22.10
- Browser: Firefox 108
- FreshRSS version: edge
- Database version: SQLite
- PHP version: 8.0.26
- Installation type: Docker Alpine

### PR DESCRIPTION
**Pull request by math-GH**
Closes #4984

Changes proposed in this pull request:

- cleaned the CSS. `content:` is just set by `frss.css` now

How to test the feature manually:

1. go to feed management (no parenthesis)
2. go to global view (parenthesis for unread articles)

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

File: p/themes/Nord/nord.css [hunk 1]
```diff
@@ -754,7 +750,6 @@ li.item.active {
 /*===========*/
 .category .title.error::before {
 	color: var(--accent-light);
-	content: "⚠ ";
```
Comment (Frenzie) [inline p/themes/Nord/nord.css:757]: This stuff is already set in frss.css I assume? :-)

**Review by Frenzie — APPROVED** [review]
lgtm

File: p/themes/Nord/nord.css (hunk 1 above)
Comment (math-GH) [reply p/themes/Nord/nord.css:757]: yes, it is:

https://github.com/FreshRSS/FreshRSS/blob/af8480651dea478e2a60dc13b9ea44d364d0f7b7/p/themes/base-theme/frss.css#L1810-L1815

**Comment by math-GH** [comment]
The unread articles counter bubbles are now in global view improved
grafik

**Pull request merged**